drivers/samsung/displays/mdc_protocol_spec.cr

```diff
@@ -0,0 +1,34 @@
+  # [header, command, id, data.size, [data], checksum]
+
+DriverSpecs.mock_driver "Samsung::Displays::MDCProtocol" do
+  id = "\x00"
+
+  # connected -> do_poll
+  # power? will take priority over status as status has priority = 0
+  # power? -> panel_mute
+  should_send("\xAA\xF9#{id}\x00\xF9")
+  responds("\xAA\xFF#{id}\x03A\xF9\x00\xFF")
+  status[:power].should eq(true)
+  # status
+  should_send("\xAA\x00#{id}\x00\x00")
+  responds("\xAA\xFF#{id}\x09A\x00\x01\x06\x00\x14\x00\x00\x00\xFF")
+  status[:hard_off].should eq(false)
+  # status[:power].should eq(true)
+  status[:volume].should eq(6)
+  status[:audio_mute].should eq(false)
+  status[:input].should eq("Vga")
+
+  exec(:volume, 24)
+  should_send("\xAA\x12#{id}\x01\x18\x12")
+  responds("\xAA\xFF\x00\x03A\x12\x18\xFF")
+  status[:volume].should eq(24)
+  status[:audio_mute].should eq(false)
+
+  exec(:mute, true)
+  responds("\xAA\xFF#{id}\x03A\xF9\x01\xFF")
+  status[:power].should eq(false)
```

Comment:
Why is power state being tested with the `mute` spec?

Reply:
I think the main reasoning behind this is because `self[:power]` is representative of whether video is actually being displayed on the tv/projector. Pretty sure this was an intentional design choice @stakach made according to these comments.

https://github.com/PlaceOS/drivers/blob/95f64ba271b69c74ebc321d2e158dda0b42fcaf7/drivers/samsung/displays/mdc_protocol.cr#L98
https://github.com/PlaceOS/drivers/blob/95f64ba271b69c74ebc321d2e158dda0b42fcaf7/drivers/samsung/displays/mdc_protocol.cr#L104-L105

Reply:
which is why `def power` uses `Command::Panel_Mute`